ng-dev/release/stamping/env-stamp.ts

```diff
@@ -45,41 +46,37 @@ function hasLocalChanges(git: GitClient) {
 }
 
 /**
- * Get the versions for generated packages.
+ * Get the versions for generated packages. The stamped versions are always based
+ * on the workspace version. Relying on tags is less reliable because tags can be
+ * modified easily in an untracked/uncontrolled way, and are less predictable with
+ * regards to the source control revision currently being checked out.
+ *
+ * A concrete use-case: The release tool tags the versioning commit only after building
+ * and publishing to NPM, causing snapshot-docs deployment to display versions from
+ * a previous version because the CI push for the bump commits executes earlier.
  *
- * In snapshot mode, the version is based on the most recent semver tag.
- * In release mode, the version is based on the workspace version.
+ * In snapshot mode, we will include the current SHA along with the workspace version.
  */
 function getSCMVersions(
   git: GitClient,
   mode: EnvStampMode,
 ): {version: string; experimentalVersion: string} {
-  if (mode === 'release') {
-    const workspaceVersion = getVersionFromWorkspacePackageJson(git);
+  const workspaceVersion = getVersionFromWorkspacePackageJson(git).format();
```

Comment:
nit: this could just be `version` and `experimentalVersion` since its always the workspace and is the only version we are working with.

Reply:
👍